From c3d71391511d535ccc79d89bc97a11d3210c12d5 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Thu, 27 Apr 2023 13:18:05 +0800
Subject: [PATCH 37/68] Revert "drm/mediatek: dsi: Reduce the time of dsi from
 LP11 to sending cmd"

This reverts commit da5fd53999335be8296410b41304457788a4b1cf.
---
 drivers/gpu/drm/mediatek/mtk_dsi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/mediatek/mtk_dsi.c b/drivers/gpu/drm/mediatek/mtk_dsi.c
index a6e71b7b69b83..146c4d04f572d 100644
--- a/drivers/gpu/drm/mediatek/mtk_dsi.c
+++ b/drivers/gpu/drm/mediatek/mtk_dsi.c
@@ -704,7 +704,7 @@ static void mtk_dsi_lane_ready(struct mtk_dsi *dsi)
 		mtk_dsi_clk_ulp_mode_leave(dsi);
 		mtk_dsi_lane0_ulp_mode_leave(dsi);
 		mtk_dsi_clk_hs_mode(dsi, 0);
-		usleep_range(1000, 3000);
+		msleep(20);
 		/* The reaction time after pulling up the mipi signal for dsi_rx */
 	}
 }
-- 
2.25.1

